python/hail/expr/functions.py

```diff
@@ -1882,7 +1882,7 @@ def is_transversion(ref, alt) -> BooleanExpression:
     >>> hl.is_transition('A', 'T').value
     True
 
-    >>> hl.is_transition('A', 'G').value
+    >>> hl.is_transition('AAA', 'AGA').value
```

Comment:
not that this is a bug from this PR, but while you're in there: these (this and the one above) should say `is_transversion`. can you fix?